docs/unified/msi/faq.md

```diff
@@ -29,14 +29,6 @@ Yes, during test procedures the Dasharo firmware work has been confirmed for
 the DDR4 WIFI platform. In turn, Dasharo working correctness on
 `MSI PRO Z690-A DDR4` has been confirmed by the community.
 
-## Can I safely try this on my board?
-
-If you are afraid of bricking the board and have no means of [recovering from
-failed installation](recovery.md), we **do not recommend** trying it out until
-at least the v1.0.0 is released. Recovery process is not suitable for
-inexperienced users right now. We will be trying to make it easier, and if that
-happens, we may change our recommendation here.
```

Comment:
I think we should leave this question and rewrite it to tell about FlashBIOS. So right now it is safe to try, because no special equipment is needed to go back to stock firmware.

Reply:
@miczyg1 tried to address that here: 8483c022f524

Reply:
@miczyg1 ?

Reply:
@miczyg1 ping v3?

Reply:
@miczyg1 ping v4?